### ISSUE TITLE
Extra data

### ISSUE THREAD

**Issue by adwardlee** [ZhengdiYu/Arbitrary-Hands-3D-Reconstruction#5]
Hi @ZhengdiYu ，

As to the pretrained weights, did you use extra data to train the model? Thanks.

**Comment by ZhengdiYu** [comment]
> Hi @ZhengdiYu ，
> 
> As to the pretrained weights, did you use extra data to train the model? Thanks.

Yes, for the in-the-wild weights, we used other extra in-the-wild data for better application use.

**Comment by ZhengdiYu** [comment]
FreiHand, and some other data we manually collect with mediapipe landmarks annotation as weak-supervision.
